src/plugins/intel_npu/src/compiler/src/driver_compiler_adapter.cpp

```diff
@@ -41,46 +41,46 @@ LevelZeroCompilerAdapter::LevelZeroCompilerAdapter(std::shared_ptr<IEngineBacken
 
     switch (graphExtVersion) {
     case ZE_GRAPH_EXT_VERSION_1_3:
```

Comment:
What about Graph Extension 1.9? The plan is that driver pushes the change with 1.9 and drop the previous one from Driver extension properties list. It means that 1.8 is not expose any longer by driver and 1.9 is available together with 1.2 up to 1.7 graph extensions

Reply:
That change is not available in the graph-ext version we use in this PR. We agreed to leave that for another PR, where we would also fix the total_mem_size property.

Reply:
Thanks for explanation